README.md

```diff
@@ -2,79 +2,74 @@
 
 > **General disclaimer:** Please be aware that this template is in public preview. Therefore, expect smaller bugs and issues when working with the solution. Please submit an Issue in GitHub if you come across any issues that you would like us to fix.
 
-# Description
-The Data Management template is, as the name suggessts, classified as a management function and is at the heart of the Enterprise Scale Analytics platform. It is responsible for the governance of the platform and enables communication to ingest data sources from Azure, third-party clouds and on-premises data sources.
+## Description
+
+The Data Management template is, as the name suggests, classified as a management function and is at the heart of the **Enterprise Scale Analytics** platform. It is responsible for the governance of the platform and enables communication to ingest data sources from Azure, third-party clouds and on-premises data sources.
 
 ## What will be deployed?
 
-By default, all the services which come under Data Management Zone are enabled, and you must explicitly disable services that you don't want to be deployed. 
+By default, all the services which come under Data Management Zone are enabled, and you must explicitly disable services that you don't want to be deployed.
 
-> Note: Before deploying the resources, we recommend to check registration status of the required resource providers in your subscription. For more information, see [Resource providers for Azure services](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types).
+> [!IMPORTANT] Before deploying the resources, we recommend to check registration status of the required resource providers in your subscription. For more information, see [Resource providers for Azure services](https://docs.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types).
 
-<p align="center">
-  <img src="./docs/media/DataHub.png" alt="Data Management Zone" width="500"/> 
-</p>
+![Data Management Zone](./docs/images/DataHub.png)
```

Comment:
@xigyenge agree, with @marvinbuss we can for now put the HTML back in for all the images. When we have a tech writer onboard they can alter this back for us.